sqs.tf

```diff
@@ -0,0 +1,29 @@
+resource "aws_sqs_queue" "keycloak_to_alerts_concierge_user_updates" {
+  name = "keycloak-${var.environment}-alerts-concierge-user-updates"
+  sqs_managed_sse_enabled = true
+}
+
+# allow Keycloak to publish messages to SQS
+resource "aws_sqs_queue_policy" "keycloak_to_alerts_concierge_user_updates" {
+  queue_url = aws_sqs_queue.keycloak_to_alerts_concierge_user_updates.id
+  policy    = data.aws_iam_policy_document.keycloak_to_alerts_concierge_user_updates.json
+}
+resource "aws_iam_role_policy" "keycloak_to_alerts_concierge_user_updates_publish" {
+  name   = "keycloak-${var.environment}-alerts-concierge-user-updates-publish"
+  role   = resource.keycloak-service.keycloak-service.id
+  policy = data.aws_iam_policy_document.keycloak_to_alerts_concierge_user_updates_publish.json
```

Comment:
Are you sure both the SQS queue and the Keycloak IAM role will need an identical policy statement?

Reply:
Good catch; I think I was confusing myself looking at too many different examples. I believe that `aws_sqs_queue_policy` is the one that can go away, since Keycloak needs permission to write to the SQS queue, but the queue doesn't need any permissions itself.